db/seeds.rb

```diff
@@ -7,3 +7,141 @@
 #   ["Action", "Comedy", "Drama", "Horror"].each do |genre_name|
 #     MovieGenre.find_or_create_by!(name: genre_name)
 #   end
+
+require 'faker'
+
+# API or website (scrape) with street address, postcode and a deescription
+
+puts "Cleaning user database"
+User.destroy_all
+
+puts "Seeding user DB"
+test_user = User.new(
+  email: "test@test.com",
+  password: "123456",
+  username: "magnus-jr"
+)
+test_user.save
+
+
+User.all.each do |user|
```

Comment:
This is not necessary. Users don't have profile photos. Also, this is using the cloudinary syntax which is not valid in this case.